marine/marine.py

```diff
@@ -18,6 +18,14 @@ class MarineResult(Structure):
 MARINE_NAME = "libmarine.so"
 
 
+class MarineValidationResult(NamedTuple):
```

Comment:
I feel this class should actually be 2 classes: `MarineFilterValidationResult` and `MarineFieldsValidationResult`.

I think that returning a list of errors when validating filters is a bit overkill since the list (by definition) will only have 1 item. What do you think?

Reply:
As far as I recall, the old version of marine returns a `bool` and a `list` and I wanted to remain compatible.
Will make sure tomorrow.

Reply:
Checked.
Current version returns a `bool` and no error, so I took your advice and separated the classes :)